src/passes/OptimizeInstructions.cpp

```diff
@@ -1767,9 +1766,6 @@ struct OptimizeInstructions
       // the type received by our parent.
       std::vector<Expression*> items;
       items.push_back(builder.makeDrop(curr->ref));
-      if (curr->rtt) {
-        items.push_back(builder.makeDrop(curr->rtt));
-      }
       items.push_back(builder.makeRefNull(intendedType));
```

Comment:
We can just create a block here directly. In fact since it's just 2 items we can use `builder.makeSequence`.